microsoft/testsuites/core/azure_image_standard.py

```diff
@@ -83,10 +83,8 @@ class AzureImageStandard(TestSuite):
     # [WARNING]: Running ['tdnf', '-y', 'upgrade'] resulted in stderr output.
     # cloud-init[958]: photon.py[ERROR]: Error while installing packages
     _ERROR_WARNING_pattern: List[Pattern[str]] = [
-        re.compile(r"^(.*\[ERROR\]*)$", re.MULTILINE),
-        re.compile(r"^(.*ERROR:*)$", re.MULTILINE),
-        re.compile(r"^(.*\[WARNING\]*)$", re.MULTILINE),
-        re.compile(r"^(.*WARNING:*)$", re.MULTILINE),
+        re.compile(r"^(.*\[ERROR\]:.*)", re.MULTILINE),
```

Comment:
Please add examples above, which error want or unwanted. The previous pattern captured patterns like `ERROR: aaa`. But they are removed. Please specify the reasons.